### DISCUSSION TITLE
philips hue colour bulbs not acting as routers for aqara devices

### DISCUSSION THREAD

**Discussion by ex18a** [Koenkk/zigbee2mqtt#19469] [Devices]
### What happened?

![image](https://github.com/Koenkk/zigbee2mqtt/assets/119137845/755c69b7-185b-438c-b1ca-69da40e217cc)
i was trying to pair my aqara motion sensor (livingroom motion) to the nearest hue light (livingroom light) using the permit join button with a selected bulb (hue white and color) but would not join, it does how ever join to the philips hue white only bulbs i have. 

so i think its only the colour bulbs will not act as routers to aquara devices. im not sure if this is zigbee2mqtt issue or something else but to me that seems like a bug to me.

i noticed all my other aqura sensors have linked either to my dining room light or bathroom light both being white only bulbs or the controller usb stick plugged in the server

### What did you expect to happen?

i wanted to join a aqara motion sensor to the nearest bulb to improve routing

### How to reproduce it (minimal and precise)

try to add aqara motion sensor via a philips hue white and color bulb it will not join.
try the same with a philps hue white only bulb and it joins instantly.

### Zigbee2MQTT version

    1.33.1 commit: 7e63039

### Adapter firmware version

    20210708

### Adapter

usb-Silicon_Labs_Sonoff_Zigbee_3.0_USB_Dongle_Plus

**Comment by Koenkk** [comment]
Z2M does not control how device connect with each other, therefore this is not a Z2M issue, I will converter it to a discussion.
